middleware/lib/sfMiddleware.ts

```diff
@@ -188,25 +183,24 @@ export function applySfFnMiddleware(request: any, logger: Logger): Array<any> {
         //change the logger level, so any subsequent user function's logger.debug would log as well
         logger.setLevel(LoggerLevel.DEBUG);
         logger.debug('debug raw request in middleware');
-        logger.debug(request);
+        logger.debug(`headers=${JSON.stringify(headers)}`);
```

Comment:
Logs accessToken?  C2C JWT?

Reply:
Right now this is protected by either a *secret* `sf-debug.DEBUG=true` or environment `DEBUG=true`, neither of which are easy for user functions to set -- we only do this logging for support purposes if the user actually sets either of those settings (and there's no mechanism for environment settings in evergreen today - could probably do so in a custom buildpack).